src/components/molecules/Results.vue

```diff
@@ -36,17 +38,54 @@ export default defineComponent({
     :headers="headers"
     :items="searchResults"
     class="resultsTable"
+    calculate-widths
   >
     <!-- eslint-disable-next-line -->
      <template #item.id-name="{item}">
       <div
         class="text-truncate"
```

Comment:
Could we change this to have the ellipsis in the middle?
Can do something like this:
https://stackoverflow.com/questions/831552/ellipsis-in-the-middle-of-a-text-mac-style
I would prefer to make the change on the `searchResults` object itself though since we might lose some search functionality if we mess with the table after load

Reply:
Can do. Changed in 840b6c5.